tests/ctst/world/Zenko.ts

```diff
@@ -58,38 +137,40 @@ export default class Zenko extends World {
      * @constructor
      * @param {Object} options - parameters provided as a CLI parameter when running the tests
      */
-    constructor(options: any) {
+    constructor(options: IWorldOptions) {
         super(options);
 
         // store service users credentials from world parameters
-        if (this.parameters.ServiceUsersCredentials) {
-             Object.entries(JSON.parse(this.parameters.ServiceUsersCredentials))
-                .forEach((entry : [string, any]) => {
-                    Zenko.serviceUsersCredentials[entry[0] as string] = {
-                        AccessKeyId: entry[1].accessKey as string,
-                        SecretAccessKey: entry[1].secretKey as string,
-
+        if ((this.parameters as Parameters).ServiceUsersCredentials) {
+            Object.entries(JSON.parse(
+                (this.parameters as Parameters).ServiceUsersCredentials as string) as UserCredentials)
+                .forEach(entry => {
+                    Zenko.serviceUsersCredentials[entry[0]] = {
+                        AccessKeyId: (entry[1] as {accessKey: string}).accessKey,
+                        SecretAccessKey: (entry[1] as {secretKey: string}).secretKey,
                     };
                 });
         }
 
+        const worldParameters = this.parameters as Parameters;
         // Workaround to be able to access global parameters in BeforeAll/AfterAll hooks
-        CacheHelper.parameters = this.parameters;
-        this.cliMode.parameters = this.parameters;
-
-        if (this.parameters.AccountSessionToken) {
-            CacheHelper.ARWWI[CacheHelper.AccountName] = {
-                AccessKeyId: this.parameters.AccountAccessKey,
-                SecretAccessKey: this.parameters.AccountSecretKey,
-                SessionToken: this.parameters.AccountSessionToken,
+        CacheHelper.parameters = worldParameters;
+        const cacheParameters = CacheHelper.parameters as Parameters;
+        this.cliMode.parameters = worldParameters as ClientOptions;
```

Comment:
This looks a bit too verbose to be properly typed, all these variables should already be of the right type, otherwise, casting might lead to missed transpiler errors, as it "changes" the type (with some checks). What is the reason the `worldParameters` is created as `Parameters` and then casted to `ClientOptions`?

Here, we create three variables out of one, with different types, which is hard to read.